demos/try16_Samples_funvals_stat.py

```diff
@@ -0,0 +1,26 @@
+# Demonstrate computation of Samples statistics on both function values and parameter values.
+# %%
+import sys
+sys.path.append("..")
+import cuqi
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+samples = cuqi.samples.Samples(np.random.rand(10, 15), geometry = cuqi.geometry.MappedGeometry(cuqi.geometry.Continuous1D(10), map=lambda x:x**2))
+
+# Compute statistics on parameter values
+samples.plot_ci(95, plot_par = True) #plot in parameter space
```

Comment:
If the geometry is changed to
```
samples = cuqi.samples.Samples(3*np.random.rand(imsize**2, 15), geometry = cuqi.geometry.Continuous2D(grid=(4,4)))
```
we get
![image](https://user-images.githubusercontent.com/9976072/187635144-ec49ba46-a436-46bc-b18f-d9e4762c8aa2.png)
which appears to use parameters for the mean but functional values for the credibility interval. I know we discussed this PR would only deal with 1D-like geometry and I am unsure whether Continuous2D is included, but it seems unintuitive that `plot_par` only applies to the mean part. I know that `.funvals` does not yet work for 2D geometries, but 
```
samples.plot_ci(95)
```
works as I would expect giving all function value plots, so I wonder if the `plot_par` version can be made to plot all in parameter style in this PR?

Reply:
Thank you for investigating this! This is not an expected behavior (and it is related to the `plot_par` flag). This PR did not change this behavior (as this PR is concerned with computing statistics on function values) but I think it is better to fix it now rather than later. So I added one line change and it seems working is expected now: when  `plot_par` is True, no 2D images will appear.